web/src/core/adapters/s3Client/s3Client.ts

```diff
@@ -606,20 +606,25 @@ export function createS3Client(
 
             const { awsS3Client } = await getAwsS3Client();
 
-            await awsS3Client.send(
-                new (await import("@aws-sdk/client-s3")).DeleteObjectsCommand({
-                    Bucket: bucketName,
-                    Delete: {
-                        Objects: paths.map(path => {
-                            const { objectName } =
-                                bucketNameAndObjectNameFromS3Path(path);
-                            return {
-                                Key: objectName
-                            };
-                        })
-                    }
-                })
-            );
+            try {
+                await awsS3Client.send(
+                    new (await import("@aws-sdk/client-s3")).DeleteObjectsCommand({
```

Comment:
The dynamic import could reject (same reasoning), move outside the try/catch block.